docs/sdks/swift-sdk/integrate-with-auth0.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Integrate With Auth0
+sidebar_position: 3
+---
+
+This guide describes how to configure Auth0 to delegate to Beyond Identity for authentication during an OAuth2 authorization flow. 
+
+### Prerequisites
+
+ - [Integrate With Auth0](/guides/sso-integrations/integrate-with-auth0)
+ - [Swift SDK Overview](overview)
+
+Before calling [`Embedded.shared.authenticate`](overview#authentication), we must [Authorize With Auth0](#authorize-with-auth0).
+
+### Authorize With Auth0
+
+ - Step 1: Configuring the Authenticator Config
+
+Make sure the [Authentication Config](/docs/v1/platform-overview/authenticator-config#embedded) in the Beyond Identity Console is set to type `Embedded` and that the Invoke URL points to your application with either an App Scheme or a Universal Link.
```

Comment:
authentication -> authenticator

Reply:
Sure, I'll update this in all the places